packages/vscode/src/plugins/studio/index.ts

```diff
@@ -0,0 +1,100 @@
+import * as cp from 'child_process'
+import * as vscode from 'vscode'
+import { getResourceURI } from '../../util'
+import { PrismaVSCodePlugin } from '../types'
+
+let studioProcess: cp.ChildProcessWithoutNullStreams
+
+const plugin: PrismaVSCodePlugin = {
+  name: 'studio',
+  enabled: () => true,
+  commands: [
+    {
+      id: 'prisma.plugin.studio.kill',
+      action: () => {
+        if (studioProcess) {
+          studioProcess.kill()
+        }
+      },
+    },
+    {
+      id: 'prisma.plugin.studio.open',
+      action: (context) => {
+        if (!vscode.workspace.rootPath) return
+        const options: cp.SpawnOptionsWithoutStdio | undefined = {
+          cwd: vscode.workspace.rootPath,
+          env: {
+            ...process.env,
+            BROWSER: 'none',
+          },
+        }
+
+        studioProcess = cp.spawn(
```

Comment:
Might be better to use `execa`here  I think (https://github.com/sindresorhus/execa)

Reply:
I did think about it but I need to read the stdout live https://stackoverflow.com/questions/10232192/exec-display-stdout-live